### PR TITLE
Add Generating Objects Section & Plugins Section

### PR DESCRIPTION
**Pull request by SooKim1110**
## Summary
Adding Generating Objects Section & Plugins Section for Fixture Monkey documentation version 2.

## (Optional): Description
- Adding generating-objects section
- Adding plugins section (jakarta-validation, kotlin, jackson)
- Adding intellij plugin section

To view the new document in your local environment,
checkout to the branch and run npm install, followed by npm run

### CODE REVIEW

File: content/en/docs/generating-objects/fixture-monkey.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+{{< tabpane persist=false >}}
+{{< tab header="Java" lang="java">}}
+
+FixtureMonkey fixtureMonkey = FixtureMonkey.create();
+
+{{< /tab >}}
+{{< tab header="Kotlin" lang="kotlin">}}
+
+val fixtureMonkey = FixtureMonkey.create()
```
Comment (acktsap) [inline content/en/docs/generating-objects/fixture-monkey.md:23]: How about providing guide using `KotlinPlugin()` as it's a required in the Kotlin env

File: content/en/docs/generating-objects/fixture-monkey.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+Product product = new Product(1L, "Book", ...);
+
+ArbitraryBuilder<Product> productBuilder = fixtureMonkey.giveMeBuilder(product);
+
+{{< /tab >}}
+{{< tab header="Kotlin" lang="kotlin">}}
+
+val product = Product(1L, "Book", ...)
+
+ArbitraryBuilder<Product> productBuilder: ArbitraryBuilder<Product> = fixtureMonkey.giveMeBuilder(product)
```
Comment (acktsap) [inline content/en/docs/generating-objects/fixture-monkey.md:139]: Exact type is used in kotlin example. Maybe `val` should be used

File: content/en/docs/generating-objects/fixture-monkey.md [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+Arbitrary<Product> productArbitrary = fixtureMonkey.giveMeArbitrary(Product.class);
+
+Arbitrary<List<String>> strListArbitrary = fixtureMonkey.giveMeArbitrary(new TypeReference<List<String>>() {});
+
+{{< /tab >}}
+{{< tab header="Kotlin" lang="kotlin">}}
+
+val productArbitrary: Arbitrary<Product> = fixtureMonkey.giveMeArbitrary()
+
+val strListArbitrary: Arbitrary<List<String>> =fixtureMonkey.giveMeArbitrary()
```
Comment (acktsap) [inline content/en/docs/generating-objects/fixture-monkey.md:203]: No space after `=`

File: content/en/docs/plugins/jackson-plugin/jackson-object-arbitrary-introspector.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+weight: 20
+---
+
+## JacksonObjectArbitraryIntrospector
+The `JacksonObjectArbitraryIntrospector` becomes the default introspector when the Jackson plugin is added.
+It puts the created properties of the given class into a map and deserializes them using Jackson's object mapper.
+
+**Example Java Class :**
+```java
+@Value // lombok getter, setter
```
Comment (acktsap) [inline content/en/docs/plugins/jackson-plugin/jackson-object-arbitrary-introspector.md:17]: I think `setter` comment should be removed as `@Value` annotation does not generates setter.

File: content/en/docs/plugins/jackson-plugin/jackson-object-arbitrary-introspector.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+public class Product {
+    private long id;
```
Comment (acktsap) [inline content/en/docs/plugins/jackson-plugin/jackson-object-arbitrary-introspector.md:19]: When using `@Value` annotation, `private` can be omitted.

File: content/en/docs/plugins/jackson-plugin/jackson-annotations.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+    .build();
+
+// when
+Product actual = fixtureMonkey.giveMeBuilder(Product.class)
+    .set("name", "book")
+    .sample();
+
+// then
+then(actual.getProductName()).isEqualTo("book"); // @JsonProperty
+then(actual.getOptions()).isNull();  // @JsonIgnore
```
Comment (acktsap) [inline content/en/docs/plugins/jackson-plugin/jackson-annotations.md:54]: No indent in the method body.

File: content/en/docs/plugins/jackson-plugin/jackson-annotations.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+}
+```
```
Comment (acktsap) [inline content/en/docs/plugins/jackson-plugin/jackson-annotations.md:56]: No kotlin example here?

File: content/en/docs/plugins/kotlin-plugin/features.md [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+  <groupId>com.navercorp.fixturemonkey</groupId>
+  <artifactId>fixture-monkey-kotlin</artifactId>
+  <version>0.6.2</version>
+  <scope>test</scope>
+</dependency>
+```
+
+##### fixture-monkey-starter-kotlin
+
+To help you get started using Fixture Monkey in a Kotlin environment, there is also a starter dependency **fixture-monkey-kotlin-starter** that comes with pre-configured dependencies such as lombok or fixture-monkey-jakarta-validation.
```
Comment (acktsap) [inline content/en/docs/plugins/kotlin-plugin/features.md:35]: I think lombok should be removed. It looks to do not provides lombok dependencies.

Comment (acktsap) [reply content/en/docs/plugins/kotlin-plugin/features.md:35]: Screen Shot 2023-09-12 at 1 36 52

File: content/en/docs/plugins/kotlin-plugin/features.md [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+  <groupId>com.navercorp.fixturemonkey</groupId>
+  <artifactId>fixture-monkey-starter-kotlin</artifactId>
+  <version>0.6.2</version>
+  <scope>test</scope>
+</dependency>
+```
+
+## Plugin
+{{< tabpane persist=false >}}
+{{< tab header="Java" lang="java">}}
```
Comment (acktsap) [inline content/en/docs/plugins/kotlin-plugin/features.md:54]: Should be include java exmaple?

**Comment by SooKim1110** [comment]
@acktsap, Thanks for taking the time to review the documentation! I've made all the changes you mentioned, but please let me know if I've missed anything!

**Comment by SooKim1110** [comment]
I will merge this PR in order to proceed with the next task. If you have any comments, please provide them in the next PR.

**Pull request merged**